.lintr

```diff
@@ -1,5 +1,5 @@
 linters:
   linters_with_defaults(
-    line_length_linter = line_length_linter(100),
+    line_length_linter = line_length_linter(120),
     object_usage_linter = NULL  # Does not work with `box::use()`.
   )
```

Comment:
I suggest adding an exclusion based on the line or using `# nolint line_length_linter` at the end of the line

```suggestion
    line_length_linter = line_length_linter(100),
    object_usage_linter = NULL  # Does not work with `box::use()`.
  )
exclusions:
  list(
    "app/view/user_guide.R" = 114
  )

```

Reply:
Thanks André. I was thinking we can have a global length of 120 to have some more breathing space. But looks like only one place exceeds the 100-character limit so I added the exception locally.